src/components/ShowPoster.tsx

```diff
@@ -222,7 +222,7 @@ const ShowPoster: React.FC<ShowPosterProps> = ({
             className={`m-1 flex w-[180px] rounded-sm ${containerStyleOverride}`}
         >
             <ShowPosterButtons visible={hover} details={details} {...rest} />
-            <Link to={`/details/${details.media_type}/${details.id}`} state={details}>
```

Comment:
This change should not be needed. If you add the `convertResultsToShowType` method to the end of the `getMovieInTheater` function this should be defined.